integration-requirements.txt

```diff
@@ -10,3 +10,4 @@ pycloudlib==1!3.0.2
 pytest<=7.3.1
 
 packaging
+passlib
```

Comment:
Nit: Is this actually needed? AFAIK, integration requirements get installed on the host system, not on the target where tests actually run.

Reply:
> Is this actually needed?

Yes - `tests/integration_tests/bugs/test_gh671.py` imports it on the host system.